packages/forms/src/elements/MultiThumbRange.js

```diff
@@ -7,101 +7,37 @@
 
 import React, { useState, useEffect, useCallback, useRef } from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
-import styled from 'styled-components';
 import debounce from 'lodash.debounce';
 import { KEY_CODES } from '@zendeskgarden/container-utilities';
+import { withTheme, isRtl, getDocument } from '@zendeskgarden/react-theming';
```

Comment:
Could we reference the theme directly here instead of using the `isRtl` helper? Same with `getDocument`

Reply:
Removed `isRtl`; left `getDocument` as it guarantees a valid document (which does not exist in the default theme).